pkgs/development/python-modules/pillowfight/default.nix

```
@@ -16,7 +16,7 @@ buildPythonPackage rec {
   propagatedBuildInputs = [ pillow ];
 
   meta = with stdenv.lib; {
-    description = "Pillow Fight";
+    description = "Eases the transition from PIL to Pillow for Python packages.";
```

https://nixos.org/manual/nixpkgs/stable/#sec-standard-meta-attributes
```suggestion
    description = "Eases the transition from PIL to Pillow for Python packages";
```

---

Thanks, fixed.